SwiftVoiceCallKitQuickstart/ViewController.swift

```diff
@@ -247,15 +268,19 @@ class ViewController: UIViewController, PKPushRegistryDelegate, TVONotificationD
         self.callKitCompletionCallback = nil
         
         stopSpin()
-        toggleUIState(isEnabled: true)
+        toggleUIState(isEnabled: true, showCallControl: false)
         self.placeCallButton.setTitle("Call", for: .normal)
     }
     
     
     // MARK: AVAudioSession
-    func routeAudioToSpeaker() {
+    func routeAudioToSpeaker(toSpeaker: Bool) {
         do {
-            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayAndRecord, with: .defaultToSpeaker)
+            if (toSpeaker) {
+                try AVAudioSession.sharedInstance().overrideOutputAudioPort(.speaker)
+            } else {
+                try AVAudioSession.sharedInstance().overrideOutputAudioPort(.none)
+            }
```

Comment:
Add some documentation here?  something like .. "The mode set by voice SDK is  "VoiceChat" so the default audio route is in-built receiver. Using port override to switch the route"